### DISCUSSION TITLE
How/where are icons generated?

### DISCUSSION THREAD

**Discussion by TehPeGaSuS** [PrivateBin/PrivateBin#1056] [Q&A]
I want to host a PrivateBin instance but, while I'd keep all links to the project, I'd like to serve my own set of icons.

How do you create/generate them? Is there any website for that matter?

TIA

**Comment by elrido** [comment]
I'm not sure if I understand the question correctly. Are you asking how to [modify the template](https://github.com/PrivateBin/PrivateBin/wiki/Templates)? If you create your own, you can change the icons via a custom CSS file or directly add yours in the HTML, like the tpl/page.php does.

**Comment by TehPeGaSuS** [comment]
Not exactly.

Nonetheless, I've replaced managed to replace the icons on my instance located at [https://0bin.xyz](https://0bin.xyz) and all seems to work as expected.

Thanks for this amazing piece of software :D
